spec/features/user/account/cancel_spec.rb

```diff
@@ -1,21 +1,21 @@
 require "rails_helper"
 
-feature "Cancel account" do
+feature "Cancel Account" do
   let(:user) { create :user, :confirmed }
 
-  before(:each) do
-    sign_in(user.email, "123456")
+  before do
+    login_as user
```

Comment:
@timurvafin, I consider `login_as` as a part of integration tests DSL similar to Capybara DSL (click_link, fill_in).

As for shared contexts, I don't think that we should use them here because:

* we don't know if it actually will be used in project later
* we don't know what exact shared contexts (only user? Or user + account + role in account?) will be used later